packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -1068,6 +1069,11 @@ class VirtualListBasic extends Component {
 					itemRefs.current[key] = (parseInt(itemNode.dataset.index) === index) ?
 						itemNode :
 						ref.querySelector(`[data-index="${index}"]`);
+
+					if (itemRefs.current[key]) {
```

Comment:
Do we need this code?
I think we can just store `ref` in `this.itemContainerRefs[key]` like
this.itemContainerRefs[key] = ref;
We already know the container's ref.